src/test/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalkTest.java

```diff
@@ -47,6 +49,7 @@ public static void beforeAll() throws URISyntaxException {
         testFilesDirectory = new File(Ddm2EmdCrosswalkTest.class.getResource("/ddm2emdCrosswalk").toURI());
 
         checkTestDataConsistency(testFilesDirectory);
+        assertEquals(Charset.forName("UTF-8"), Charset.defaultCharset());
```

Comment:
Can you add an explanation of what to do when you see this error?

```suggestion
        assertEquals("please put an explanation here", Charset.forName("UTF-8"), Charset.defaultCharset());
```